rules/sql-real-world-indexes/rule.md

```diff
@@ -0,0 +1,53 @@
+---
+seoDescription: Understanding how SQL indexes work and common real world ways of visualising them
+type: rule
+archivedreason:
+title: Do You Know How to visualise SQL indexes?
```

Comment:
Hi @brydeno 

1. Can you fix the Capitalization in this title to be standard?
2. Also make it American English... "visualize" (consider fixing that on the rest of the content as well)

Reply:
Done